SourceryTests/Parsing/ComposerSpec.swift

```diff
@@ -80,6 +80,7 @@ class ParserComposerSpec: QuickSpec {
                                                                               typeName: TypeName("String"),
                                                                               defaultValue: "\"Baz\"")],
                                                  returnTypeName: TypeName("Void"),
+                                                 accessLevel: .none,
```

Comment:
These changes were required to get the tests to pass, it appears that SourceKit no longer reports access levels for extensions the same way that it did previously